bob-api/index.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+// Include the api
+include 'api/de_onderwijzers_api.php';
+
+// Enable debug mode for the API
+api\DeOnderwijzers::$DEBUG_ENABLED = true;
+
+// Create a new instance of the api
+$api = new api\DeOnderwijzers();
+
+// Try the validate credentials
+$result = $api->validateCredentials('a', 'password');
```

Comment:
in plaats van a, is het mischien duidelijker om 'username' neer te zetten

Reply:
Agreed, gaan we fixen!